docs/framework/install/on-windows-10.md

```diff
@@ -74,4 +74,4 @@ If you cannot get the correct version of the .NET Framework installed, you can [
 
 [.NET Downloads](https://www.microsoft.com/net/download?utm_source=ms-docs&utm_medium=referral)   
 [Troubleshoot blocked .NET Framework installations and uninstallations](troubleshoot-blocked-installations-and-uninstallations.md)   
-[Install the .NET Framework for developers](guide-for-developers.md)
\ No newline at end of file
+[Install the .NET Framework for developers](guide-for-developers.md)
```

Comment:
Dont understand why this was considered a changed, but its ok.

Reply:
Sometimes that happens with the last line @jzeferino